### ISSUE TITLE
Please example to use this instead of Session in Template level?

### ISSUE THREAD

**Issue by thearabbit** [xamfoo/reactive-obj#4]
Could you example me to use this instead of Session in template level

``` js
Template.myTpl.onRendered(function(){
   Session.set('value', 'MyValue');
});

Template.myTpl.helpers({
   data: function(){
         return Session.get('value');
   }
});
```

**Comment by xamfoo** [comment]
See #1

**Comment by thearabbit** [comment]
Sorry I don't understand.

**Comment by xamfoo** [comment]
Quoting from #1 

> If you want to declare a variable per template instance, an easy way is:
> 
> ``` javascript
> Template.myTem.onCreated(function(){
>    this.state = new ReactiveObj({a: 5});
> });
> 
> Template.myTem.helpers({
>    helper: function(){
>       return Template.instance().state.get('a');
>    }
> });
> ```

If this does not help could you rephrase your question?

**Comment by thearabbit** [comment]
Oh if I want to use instead of session for any where on client, I must to declare without `var`.
right or wrong?

**Comment by xamfoo** [comment]
Yes you can declare a `reactiveObj` globally and use it like `Session`:

``` javascript
AppState = new ReactiveObj;

Template.myTpl.onRendered(function(){
   AppState.set('value', 'MyValue');
});

Template.myTpl.helpers({
   data: function(){
         return AppState.get('value');
   }
});
```

**Comment by thearabbit** [comment]
Thanks again.

**Comment by thearabbit** [comment]
Excuse me, could I use it in `common folder`?

**Comment by xamfoo** [comment]
`ReactiveObj` is both a server and client side package so you should be able to put its code in a common folder.
